.idea/vcs.xml

```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="CommitMessageInspectionProfile">
```

Comment:
If we formalize these suggestions I'd like to follow https://chris.beams.io/posts/git-commit/